pkg/types/ccipocr3/config.go

```diff
@@ -58,13 +58,19 @@ func (c CommitPluginConfig) Validate() error {
 
 type ExecutePluginConfig struct {
 	// DestChain is the ccip destination chain configured for the execute DON.
-	DestChain ChainSelector
+	DestChain ChainSelector `json:"destChain"`
 
 	// ObserverInfo is a map of oracle IDs to ObserverInfo.
-	ObserverInfo map[commontypes.OracleID]ObserverInfo
+	ObserverInfo map[commontypes.OracleID]ObserverInfo `json:"observerInfo"`
 
 	// MessageVisibilityInterval is the time interval for which the messages are visible by the plugin.
-	MessageVisibilityInterval time.Duration
+	MessageVisibilityInterval time.Duration `json:"messageVisibilityInterval"`
+
+	// FChain defines the FChain value for each chain. FChain is used while forming consensus based on the observations.
+	FChain map[ChainSelector]int `json:"fChain"`
+
+	// F is the ocr3 big F value for the execute plugin.
+	F int `json:"F"`
```

Comment:
I named it this way since we've sometimes referred to this as big f and little f. I don't have a strong opinion either way, I suppose we've moved on to f and fChain